### PR TITLE
Global refresh 28 03

### PR DESCRIPTION
**Pull request by tutacrypto**

### CODE REVIEW

File: new-module/vaults/liquidations.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ To make sure collateral is always backing agTokens issued, Angle Vaults can get
 
 As we will see in the [Variable liquidations amount](liquidations.md#variable-liquidations-amount), liquidations in Angle Borrowing module have been designed to be as much borrower friendly as possble.
 
-**TLDR**
+### TL;DR
```
[the remark this answers is not in the document]
Comment (tutacrypto) [reply new-module/vaults/liquidations.md:11]: Yes, but it doesn't really make sense it's a magnifying glass when the TLDR is an overview 
I can change it everywhere though

File: new-module/README.md [hunk 2]
```diff
@@ -6,32 +6,36 @@ description: Angle Borrowing Module Overview
...
 * Easier to add new collaterals
 * Users can keep exposure to their collateral tokens while using agEUR
 
-These advantages help further expand Angle and its agTokens, both in terms of the stablecoins issued, their volume, and the chains and networks where they can be issued/redeemed. It has been designed to work hand in hand with the current one, and strenghten the Angle Protocol. AgTokens, like agEUR, are fully interoperable between both modules.
+These advantages help **further expand Angle and its agTokens**, both in terms of the stablecoins issued, their volume, and the chains and networks where they can be issued & redeemed. It has been designed to work hand in hand with the current one, and strenghten the Angle Protocol. AgTokens, like agEUR, are fully interoperable between both modules.
 
 ## Main Features
 
-### Vaults
+### 🏦 Vaults
```
[the remark this answers is not in the document]
Comment (tutacrypto) [reply new-module/README.md:21]: I added leverage and variable liquidations. 
Let me know if you have any other idea

File: concepts/stable-seekers/README.md [hunk 3]
```diff
@@ -6,24 +6,24 @@ description: Getting stablecoins from collateral and conversely
...
 
 ## 🗺️ Principle
 
-Angle is designed to be as simple as possible from a user perspective and like USDC (or also USDT), it relies on full convertibility of collateral and stable assets, meaning users can swap collateral against stablecoins as well as stablecoins against collateral at any time.
+Angle core module is designed to be as simple as possible from a user perspective and like USDC (or also USDT), it relies on full convertibility of collateral and stable assets, meaning users can swap collateral against stablecoins as well as stablecoins against collateral at any time.
 
-To generate a stable asset, a user just has to send to the protocol a whitelisted collateral. An oracle then determines how many stablecoins (`ERC-20`tokens) need to be minted and sent to the user. Like in an AMM swap, the collateral received by the protocol does no longer belong to the user.
+To generate a stable asset, a user just has to send to the core module a whitelisted collateral. An oracle then determines how many stablecoins (`ERC-20`tokens) need to be minted and sent to the user. Like in an AMM swap, the collateral received by the core module does no longer belong to the user.
```
Comment (sogipec) [inline concepts/stable-seekers/README.md:18]: the protocol

File: concepts/faq.md [hunk 4]
```diff
@@ -1,105 +1,76 @@
...
+2. To be able to always swap collateral against stablecoins and stablecoins against collateral, the core module needs to remain over-collateralized regardless of the variations of the price of the collateral: it does so by issuing perpetual futures, which are leveraging contracts. People can come to Angle, bring some collateral and choose the amount of collateral from stable seekers they want to cover: they then get leveraged with the multiplier of their choice and at the same time they insure the core module against the volatility of its collateral. If there is 1 wETH that was brought to get stablecoins by stable seekers, someone can come to Angle, bring collateral (like 0.5 wETH) and choose to cover this 1 wETH (hence getting a 3x leverage). This means that they get the capital gains or have to pay for the loss they would have realzied if they had owned this 1 wETH from the beginning.
```
Comment (sogipec) [inline concepts/faq.md:14]: realized

File: concepts/hedging-agents/README.md [hunk 5]
```diff
@@ -112,25 +112,25 @@ $$
 
 ## 🛏️ HAs Hedged Amounts
 
-When HAs enter the protocol, they specify a position size denominated in collateral, representing an amount of the protocol collateral reserves they are hedging. Yet from a protocol perspective, when HAs come in the protocol, they insure a fixed amount of stablecoins.
+When HAs enter the core module, they specify a position size denominated in collateral, representing an amount of the collateral reserves they are hedging. Yet from a core module perspective, when HAs come in, they insure a fixed amount of stablecoins.
```
[the remark this answers is not in the document]
Comment (tutacrypto) [reply concepts/hedging-agents/README.md:115]: I changed this to "contract", here and in other places where it made more sense

**Review by sogipec — APPROVED** [review]

**Pull request merged**